### DISCUSSION TITLE
How to update the error message for the InSlice

### DISCUSSION THREAD

**Discussion by khanakia** [cohesivestack/valgo#27] [Q&A] [answered]
the InSlide returns a very generic message. I want the user to show the value they can send. So the error should be like this
`Status must be one of married, single`

https://github.com/cohesivestack/valgo?tab=readme-ov-file#validationvalid-function
```
  v.String("singl", "status").InSlice([]string{"married", "single"})
```

```
{
  "status": [
    "Status is not valid"
  ]
}
```

**Comment by carlosforero** [comment, marked as answer]
You can customize the error message for `InSlice` by passing a custom template as the second parameter:

```go
val := v.Is(
    v.String("singl", "status").InSlice(
        []string{"married", "single"}, 
        "{{title}} must be one of married, single"
    )
)
```

This will output:
```json
{
  "status": [
    "Status must be one of married, single"
  ]
}
```

You can use any custom message you want as the template parameter to provide more helpful error messages to users.

If you have any feedback, don't hesitate to share it!
